public/app/features/dashboard/dashgrid/PanelChrome.tsx

```diff
@@ -90,10 +90,23 @@ export class PanelChrome extends Component<Props, State> {
       this.setState({ isFirstLoad: false });
     }
 
+    let alignmentOptions;
+
+    if (Boolean(panel.plugin?.meta.alignData) && panel.plugin?.resolveXYDimensions) {
+      alignmentOptions = {
+        withAlignment: Boolean(panel.plugin?.meta.alignData),
+        alignmentOptions: (data: DataFrame[]) => panel.plugin!.resolveXYDimensions!(data, panel.options),
+      };
```

Comment:
nitpick: the naming here feels a bit off, should the alignmentOptions function be getAlignmentMatchers ?